commit cd0def5953c39789f67fe4912823a5c11633ca6d (refs/changes/27/427/1)
Author: Orlando Vazquez <ovazquez@gmail.com>
Date:   2016-09-07T10:23:44-07:00 (3 years, 1 month ago)
    
    CNAPI-667 CNAPI dying with: TypeError: Cannot read property 'limit' of undefined

diff --git a/lib/models/waitlist.js b/lib/models/waitlist.js
index 6a92aeb..5e3f332 100644
--- a/lib/models/waitlist.js
+++ b/lib/models/waitlist.js
@@ -487,7 +487,7 @@ ModelWaitlist.query = function (filter, findOpts, callback) {
     }
 
     var findParams = jsprim.deepCopy(findOpts);
-    var sort = findParams.sort;
+    var sort = findParams.sort || {};
     findParams.sort = jsprim.deepCopy(defaultSort);
 
     if (sort.limit) {
